proxy/http/HttpConfig.h

```diff
@@ -926,8 +974,8 @@ inline HttpConfigParams::~HttpConfigParams()
   ats_free(oride.cache_vary_default_other);
   ats_free(connect_ports_string);
   ats_free(reverse_proxy_no_host_redirect);
+  ats_free(redirect_actions_string);
```

Comment:
Skimming part of the call tree from the `HttpEstablishStaticConfigStringAlloc` macro, it seems using a string type might be very tricky to do. The memory here is allocated internally by some RecCore code, and this is a pointer to that. It would take some time to change it, and it would look rather odd being the only string that is allocated in a different way than the others. If we want to go down that route we can, but it would take me some time to figure it out.

Reply:
Won't fix this here. See [comment](https://github.com/apache/trafficserver/pull/4145/files#r216478374).